### DISCUSSION TITLE
forecast solar

### DISCUSSION THREAD

**Discussion by awi69** [solectrus/solectrus#2286] [General]
Hallo,

erstmal großes Kompliment für das Dashboard ! Habe gestern installiert und auch nach Anleitung die CSVs importiert.
Jetzt weckt das Thema Solar Forecast meine "Begehrlichkeiten". Würde nach einer erfolgreichen Installation eine grafische Anzeige, wie sie auf der solar forecast seite  zu finden ist , das Dashboard zieren oder geht es hier um etwas "Anderes" ?

**Comment by ledermann** [comment]
Danke für die netten Worte :)

So ganz verstehe ich aber die Frage nicht. Das Einbinden des Forecast-Collectors holt regelmäßig die Vorhersage von https://forecast.solar für den eingestellten Standort und stellt sie in der Tagesansicht als zusätzliche graue Kurve dar. 

Beispiel hier:
https://solectrus.ledermann.dev/inverter_power/day

**Reply by awi69** [reply]
AAH ! (Stirnklatscher) jetzt sehe ich das auch ;) ... Es hat also gereicht in der . env die Daten meines Standortes sowie Ausrichtung/Dachneigung/beschattung etc. einzutragen und somit funktioniert das "out of the box". Ich hatte vermutet ich müsse den Dockerbefehl wie in solar forecast beschrieben "zusätzlich" aufrufen, jetzt wo ich nochmal das hosting tutorial bewundere, sehe ich natürlich auch den forecast container ... sorry für die dummy frage !
